pkg/writer/writer.go

```diff
@@ -74,7 +75,7 @@ func writeProjects(context *WriterContext, projectDefinitions manifest.ProjectDe
 
 		configs := collectAllConfigs(p)
 
-		errs := config.WriteConfigs(&config.WriterContext{
+		errs := writer.WriteConfigs(&writer.WriterContext{
```

Comment:
The new import paths seem pretty odd, especially in this case.
If we use readability of the public interface for design, I think a configwriter package or staying in config would make sense :/ `configwriter.Write()` / `config.Write()`.

Thinking about the loader, especially now loading a single configuration, `config.Load(...) config.Config, err` would be a very nice interface.

Reply:
IMO the package name was perfectly fine. Also, i don't see a problem with importing something that is also called writer into a package already named writer. Further, you can always alias your import to "configwriter" if needed. 

In general package names should not have underscores. (exception _test of course).